CODEOWNERS

```diff
@@ -37,6 +37,10 @@
 /crates/aptos-faucet @banool @gregnazario
 /crates/aptos-faucet-cli @gregnazario
 /crates/aptos-genesis @gregnazario
+
+# Owners for the aptos-logger crate
+/crates/aptos-logger @alexm
```

Comment:
I think we want you GitHub handle here. The PR is flagging an error with the codeowners file. Can you try `@Markuze` as that might be the correct GH handle?

Reply:
Ack